packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -139,15 +146,23 @@ describe('FocusTrapZone', () => {
 
     // Focus the first button.
     ReactTestUtils.Simulate.focus(buttonA);
-    expect(lastFocusedElement).toBe(buttonA);
+    window.requestAnimationFrame(() => {
+      expect(lastFocusedElement).toBe(buttonA);
+
+      // Pressing shift + tab should go to d.
+      ReactTestUtils.Simulate.keyDown(buttonA, { which: KeyCodes.tab, shiftKey: true });
+      window.requestAnimationFrame(() => {
+        expect(lastFocusedElement).toBe(buttonD);
 
-    // Pressing shift + tab should go to d.
-    ReactTestUtils.Simulate.keyDown(buttonA, { which: KeyCodes.tab, shiftKey: true });
-    expect(lastFocusedElement).toBe(buttonD);
+        // Pressing tab should go to a.
+        ReactTestUtils.Simulate.keyDown(buttonD, { which: KeyCodes.tab });
+        window.requestAnimationFrame(() => {
```

Comment:
hope this works in ie11. might be worth checking.

Reply:
Yup, works like a charm on the testpage